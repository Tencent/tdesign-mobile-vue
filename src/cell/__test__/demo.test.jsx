/**
 * 该文件为由脚本 `npm run test:demo` 自动生成，如需修改，执行脚本命令即可。请勿手写直接修改，否则会被覆盖
 */

import { mount } from '@vue/test-utils';
import groupVue from '@/cell/demos/group.vue';
import mobileVue from '@/cell/demos/mobile.vue';
import multipleVue from '@/cell/demos/multiple.vue';
import singleVue from '@/cell/demos/single.vue';

const mapper = {
  groupVue,
  mobileVue,
  multipleVue,
  singleVue,
};

describe('Cell', () => {
  Object.keys(mapper).forEach((demoName) => {
    it(`Cell ${demoName} demo works fine`, () => {
      const wrapper = mount(mapper[demoName]);
      expect(wrapper.element).toMatchSnapshot();
    });
  });
});
