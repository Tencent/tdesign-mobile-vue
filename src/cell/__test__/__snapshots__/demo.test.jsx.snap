// Vitest Snapshot v1

exports[`Cell > Cell borderVue demo works fine 1`] = `
<div
  class="cell-base"
>
  <div
    class="t-cell-group"
  >
    <div
      class="t-cell-group__title"
    >
      
      显示组边框
      
    </div>
    <div
      class="t-cell-group__container border--top-bottom"
    >
      
      <div
        class="t-cell t-cell--middle t-cell--bordered"
      >
        <div
          class="t-cell__left-icon"
        >
          <!--v-if-->
          <!--v-if-->
        </div>
        <div
          class="t-cell__title"
        >
          
          <div>
            自定义标题
          </div>
          
          <!--v-if-->
          <!--v-if-->
        </div>
        <div
          class="t-cell__note"
        >
          
          <div>
            内容是长长长长长长长长长长长长长长长长长长长长长的
          </div>
          
        </div>
        <!--v-if-->
      </div>
      <div
        class="t-cell t-cell--middle t-cell--bordered"
      >
        <div
          class="t-cell__left-icon"
        >
          <!--v-if-->
          <!--v-if-->
        </div>
        <div
          class="t-cell__title"
        >
          
          <div>
            自定义标题
          </div>
          
          <!--v-if-->
          <!--v-if-->
        </div>
        <div
          class="t-cell__note"
        >
          
          <div>
            内容是短的
          </div>
          
        </div>
        <!--v-if-->
      </div>
      
    </div>
  </div>
</div>
`;

exports[`Cell > Cell clickVue demo works fine 1`] = `
<div
  class="t-cell-group"
>
  <div
    class="t-cell-group__title"
  >
    
    单元格点击时触发
    
  </div>
  <div
    class="t-cell-group__container border--top-bottom"
  >
    
    <div
      class="t-cell t-cell--middle t-cell--bordered"
    >
      <div
        class="t-cell__left-icon"
      >
        <!--v-if-->
        <!--v-if-->
      </div>
      <div
        class="t-cell__title"
      >
        
        <div>
          自定义标题
        </div>
        
        <!--v-if-->
        <!--v-if-->
      </div>
      <div
        class="t-cell__note"
      >
        
        <div>
          内容是长长长长长长长长长长长长长长长长长长长长长的
        </div>
        
      </div>
      <!--v-if-->
    </div>
    <div
      class="t-cell t-cell--middle t-cell--bordered"
    >
      <div
        class="t-cell__left-icon"
      >
        <!--v-if-->
        <!--v-if-->
      </div>
      <div
        class="t-cell__title"
      >
        
        <div>
          自定义标题
        </div>
        
        <!--v-if-->
        <!--v-if-->
      </div>
      <div
        class="t-cell__note"
      >
        
        <div>
          内容是短的
        </div>
        
      </div>
      <!--v-if-->
    </div>
    
  </div>
</div>
`;

exports[`Cell > Cell groupVue demo works fine 1`] = `
<div>
  <div
    class="t-cell-group"
    summary="列表外常驻文字，常常用于辅助信息说明，列表外常驻文字，常常用于辅助信息说明"
  >
    <div
      class="t-cell-group__title"
    >
      
      分组标题
      
    </div>
    <div
      class="t-cell-group__container"
    >
      
      <div
        class="t-cell t-cell--middle t-cell--bordered"
      >
        <div
          class="t-cell__left-icon"
        >
          <!--v-if-->
          <!--v-if-->
        </div>
        <div
          class="t-cell__title"
        >
          单行标题
          <!--v-if-->
          <!--v-if-->
        </div>
        <!--v-if-->
        <!--v-if-->
      </div>
      <div
        class="t-cell t-cell--middle t-cell--bordered"
      >
        <div
          class="t-cell__left-icon"
        >
          <!--v-if-->
          <!--v-if-->
        </div>
        <div
          class="t-cell__title"
        >
          单行标题
          <!--v-if-->
          <!--v-if-->
        </div>
        <!--v-if-->
        <div
          class="t-cell__right-icon"
        >
          <svg
            class="t-icon t-icon-chevron-right"
            fill="none"
            height="1em"
            viewBox="0 0 16 16"
            width="1em"
          >
            <path
              d="M6.46 12.46l-.92-.92L9.08 8 5.54 4.46l.92-.92L10.92 8l-4.46 4.46z"
              fill="currentColor"
              fill-opacity="0.9"
            />
          </svg>
        </div>
      </div>
      
    </div>
  </div>
</div>
`;

exports[`Cell > Cell mobileVue demo works fine 1`] = `
<div
  class="tdesign-mobile-demo"
>
  <h1
    class="title"
  >
    Cell 单元格
  </h1>
  <p
    class="summary"
  >
    一行内容/功能的垂直排列方式。一行项目左侧为主要内容展示区域，右侧可增加更多操作内容
  </p>
  <div
    class="tdesign-mobile-demo-block"
  >
    <h2
      class="tdesign-mobile-demo-block__title"
    >
      01 类型
    </h2>
    <p
      class="tdesign-mobile-demo-block__summary"
    >
      单行单元格
    </p>
    
    <div>
      <div
        class="t-cell-group"
      >
        <!--v-if-->
        <div
          class="t-cell-group__container border--top-bottom"
        >
          
          <div
            class="t-cell t-cell--middle t-cell--bordered"
          >
            <div
              class="t-cell__left-icon"
            >
              <!--v-if-->
              <!--v-if-->
            </div>
            <div
              class="t-cell__title"
            >
              单行标题
              <!--v-if-->
              <!--v-if-->
            </div>
            <!--v-if-->
            <!--v-if-->
          </div>
          <div
            class="t-cell t-cell--middle t-cell--bordered"
          >
            <div
              class="t-cell__left-icon"
            >
              <!--v-if-->
              <!--v-if-->
            </div>
            <div
              class="t-cell__title"
            >
              单行标题
              <span
                class="t-cell--required"
              >
                 *
              </span>
              <!--v-if-->
            </div>
            <!--v-if-->
            <!--v-if-->
          </div>
          <div
            class="t-cell t-cell--middle t-cell--bordered"
          >
            <div
              class="t-cell__left-icon"
            >
              <!--v-if-->
              <!--v-if-->
            </div>
            <div
              class="t-cell__title"
            >
              单行标题
              <!--v-if-->
              <!--v-if-->
            </div>
            <div
              class="t-cell__note"
            >
              辅助信息
            </div>
            <!--v-if-->
          </div>
          <div
            class="t-cell t-cell--middle t-cell--bordered"
          >
            <div
              class="t-cell__left-icon"
            >
              <!--v-if-->
              <!--v-if-->
            </div>
            <div
              class="t-cell__title"
            >
              单行标题
              <!--v-if-->
              <!--v-if-->
            </div>
            <!--v-if-->
            <div
              class="t-cell__right-icon"
            >
              <svg
                class="t-icon t-icon-chevron-right"
                fill="none"
                height="1em"
                viewBox="0 0 16 16"
                width="1em"
              >
                <path
                  d="M6.46 12.46l-.92-.92L9.08 8 5.54 4.46l.92-.92L10.92 8l-4.46 4.46z"
                  fill="currentColor"
                  fill-opacity="0.9"
                />
              </svg>
            </div>
          </div>
          <div
            class="t-cell t-cell--middle t-cell--bordered"
          >
            <div
              class="t-cell__left-icon"
            >
              <!--v-if-->
              <!--v-if-->
            </div>
            <div
              class="t-cell__title"
            >
              单行标题
              <!--v-if-->
              <!--v-if-->
            </div>
            <div
              class="t-cell__note"
            >
              辅助信息
            </div>
            <div
              class="t-cell__right-icon"
            >
              <svg
                class="t-icon t-icon-chevron-right"
                fill="none"
                height="1em"
                viewBox="0 0 16 16"
                width="1em"
              >
                <path
                  d="M6.46 12.46l-.92-.92L9.08 8 5.54 4.46l.92-.92L10.92 8l-4.46 4.46z"
                  fill="currentColor"
                  fill-opacity="0.9"
                />
              </svg>
            </div>
          </div>
          <div
            class="t-cell t-cell--middle t-cell--bordered"
          >
            <div
              class="t-cell__left-icon"
            >
              <!--v-if-->
              <!--v-if-->
            </div>
            <div
              class="t-cell__title"
            >
              单行标题
              <!--v-if-->
              <!--v-if-->
            </div>
            <div
              class="t-cell__note"
            >
              
              <div
                class="t-badge"
              >
                <div
                  class="t-badge__inner t-badge--medium t-badge--circle"
                >
                  2
                </div>
                <!---->
              </div>
              
            </div>
            <div
              class="t-cell__right-icon"
            >
              <svg
                class="t-icon t-icon-chevron-right"
                fill="none"
                height="1em"
                viewBox="0 0 16 16"
                width="1em"
              >
                <path
                  d="M6.46 12.46l-.92-.92L9.08 8 5.54 4.46l.92-.92L10.92 8l-4.46 4.46z"
                  fill="currentColor"
                  fill-opacity="0.9"
                />
              </svg>
            </div>
          </div>
          <div
            class="t-cell t-cell--middle t-cell--bordered"
          >
            <div
              class="t-cell__left-icon"
            >
              <!--v-if-->
              <!--v-if-->
            </div>
            <div
              class="t-cell__title"
            >
              单行标题
              <!--v-if-->
              <!--v-if-->
            </div>
            <div
              class="t-cell__note"
            >
              
              <span
                class="t-switch t-is-checked"
              >
                <!--v-if-->
                <span
                  class="t-switch__node"
                  style=""
                />
              </span>
              
            </div>
            <!--v-if-->
          </div>
          <div
            class="t-cell t-cell--middle t-cell--bordered"
          >
            <div
              class="t-cell__left-icon"
            >
              <svg
                class="t-icon t-icon-app"
                fill="none"
                height="1em"
                viewBox="0 0 16 16"
                width="1em"
              >
                <path
                  d="M11.25 1.75a3 3 0 100 6 3 3 0 000-6zm-2 3a2 2 0 114 0 2 2 0 01-4 0zM2 3a1 1 0 011-1h3.5a1 1 0 011 1v3.5a1 1 0 01-1 1H3a1 1 0 01-1-1V3zm1 0v3.5h3.5V3H3zM2 9.5a1 1 0 011-1h3.5a1 1 0 011 1V13a1 1 0 01-1 1H3a1 1 0 01-1-1V9.5zm1 0V13h3.5V9.5H3zM8.5 9.5a1 1 0 011-1H13a1 1 0 011 1V13a1 1 0 01-1 1H9.5a1 1 0 01-1-1V9.5zm1 3.5H13V9.5H9.5V13z"
                  fill="currentColor"
                  fill-opacity="0.9"
                />
              </svg>
              <!--v-if-->
            </div>
            <div
              class="t-cell__title"
            >
              单行标题
              <!--v-if-->
              <!--v-if-->
            </div>
            <!--v-if-->
            <!--v-if-->
          </div>
          
        </div>
      </div>
    </div>
    
  </div>
  <div
    class="tdesign-mobile-demo-block"
  >
    <!--v-if-->
    <p
      class="tdesign-mobile-demo-block__summary"
    >
      多行单元格
    </p>
    
    <div>
      <div
        class="t-cell-group"
      >
        <!--v-if-->
        <div
          class="t-cell-group__container"
        >
          
          <div
            class="t-cell t-cell--middle t-cell--bordered"
          >
            <div
              class="t-cell__left-icon"
            >
              <!--v-if-->
              <!--v-if-->
            </div>
            <div
              class="t-cell__title"
            >
              多行标题
              <!--v-if-->
              <div
                class="t-cell__description"
              >
                一段很长很长的内容文字
              </div>
            </div>
            <!--v-if-->
            <!--v-if-->
          </div>
          <div
            class="t-cell t-cell--middle t-cell--bordered"
          >
            <div
              class="t-cell__left-icon"
            >
              
              <svg
                class="t-icon t-icon-app"
                fill="none"
                height="1em"
                viewBox="0 0 16 16"
                width="1em"
              >
                <path
                  d="M11.25 1.75a3 3 0 100 6 3 3 0 000-6zm-2 3a2 2 0 114 0 2 2 0 01-4 0zM2 3a1 1 0 011-1h3.5a1 1 0 011 1v3.5a1 1 0 01-1 1H3a1 1 0 01-1-1V3zm1 0v3.5h3.5V3H3zM2 9.5a1 1 0 011-1h3.5a1 1 0 011 1V13a1 1 0 01-1 1H3a1 1 0 01-1-1V9.5zm1 0V13h3.5V9.5H3zM8.5 9.5a1 1 0 011-1H13a1 1 0 011 1V13a1 1 0 01-1 1H9.5a1 1 0 01-1-1V9.5zm1 3.5H13V9.5H9.5V13z"
                  fill="currentColor"
                  fill-opacity="0.9"
                />
              </svg>
              
              <!--v-if-->
            </div>
            <div
              class="t-cell__title"
            >
              多行带图标
              <!--v-if-->
              <div
                class="t-cell__description"
              >
                一段很长很长的内容文字
              </div>
            </div>
            <!--v-if-->
            <div
              class="t-cell__right-icon"
            >
              <svg
                class="t-icon t-icon-chevron-right"
                fill="none"
                height="1em"
                viewBox="0 0 16 16"
                width="1em"
              >
                <path
                  d="M6.46 12.46l-.92-.92L9.08 8 5.54 4.46l.92-.92L10.92 8l-4.46 4.46z"
                  fill="currentColor"
                  fill-opacity="0.9"
                />
              </svg>
            </div>
          </div>
          <div
            class="t-cell t-cell--middle t-cell--bordered"
          >
            <div
              class="t-cell__left-icon"
            >
              
              <div
                class="t-avatar t-avatar--circle"
              >
                <div
                  class="t-avatar__inner"
                >
                  <div
                    class="t-image t-image--round"
                  >
                    <div
                      class="t-image__status"
                    >
                      <svg
                        class="t-icon t-icon-ellipsis"
                        fill="none"
                        height="1em"
                        viewBox="0 0 16 16"
                        width="1em"
                      >
                        <path
                          d="M3 9a1 1 0 110-2 1 1 0 010 2zM7 8a1 1 0 102 0 1 1 0 00-2 0zM12 8a1 1 0 102 0 1 1 0 00-2 0z"
                          fill="currentColor"
                          fill-opacity="0.9"
                        />
                      </svg>
                    </div>
                    <img
                      alt=""
                      class="t-image__img"
                      src="https://tdesign.gtimg.com/mobile/demos/avatar_1.png"
                      style="object-fit: fill; object-position: center;"
                    />
                  </div>
                </div>
                <!--v-if-->
              </div>
              
              <!--v-if-->
            </div>
            <div
              class="t-cell__title"
            >
              多行带头像
              <!--v-if-->
              <div
                class="t-cell__description"
              >
                一段很长很长的内容文字
              </div>
            </div>
            <!--v-if-->
            <div
              class="t-cell__right-icon"
            >
              <svg
                class="t-icon t-icon-chevron-right"
                fill="none"
                height="1em"
                viewBox="0 0 16 16"
                width="1em"
              >
                <path
                  d="M6.46 12.46l-.92-.92L9.08 8 5.54 4.46l.92-.92L10.92 8l-4.46 4.46z"
                  fill="currentColor"
                  fill-opacity="0.9"
                />
              </svg>
            </div>
          </div>
          <div
            class="t-cell t-cell--middle t-cell--bordered"
          >
            <div
              class="t-cell__left-icon"
            >
              <!--v-if-->
              
              <img
                class="t-cell__image"
                src="https://tdesign.gtimg.com/mobile/%E5%9B%BE%E7%89%87.png"
              />
              
            </div>
            <div
              class="t-cell__title"
            >
              多行带图片
              <!--v-if-->
              <div
                class="t-cell__description"
              >
                一段很长很长的内容文字
              </div>
            </div>
            <!--v-if-->
            <!--v-if-->
          </div>
          <div
            class="t-cell t-cell--middle t-cell--bordered"
          >
            <div
              class="t-cell__left-icon"
            >
              <!--v-if-->
              <!--v-if-->
            </div>
            <div
              class="t-cell__title"
            >
              多行标题
              <!--v-if-->
              <div
                class="t-cell__description"
              >
                一段很长很长的内容文字，长文本自动换行，该选项的描述是一段很长的内容
              </div>
            </div>
            <!--v-if-->
            <!--v-if-->
          </div>
          <div
            class="t-cell t-cell--middle t-cell--bordered"
          >
            <div
              class="t-cell__left-icon"
            >
              <!--v-if-->
              <!--v-if-->
            </div>
            <div
              class="t-cell__title"
            >
              多行高度不定，长文本自动换行，该选项的描述是一段很长的内容
              <!--v-if-->
              <div
                class="t-cell__description"
              >
                一段很长很长的内容文字，长文本自动换行，该选项的描述是一段很长的内容
              </div>
            </div>
            <!--v-if-->
            <!--v-if-->
          </div>
          
        </div>
      </div>
    </div>
    
  </div>
</div>
`;

exports[`Cell > Cell multipleVue demo works fine 1`] = `
<div>
  <div
    class="t-cell-group"
  >
    <!--v-if-->
    <div
      class="t-cell-group__container"
    >
      
      <div
        class="t-cell t-cell--middle t-cell--bordered"
      >
        <div
          class="t-cell__left-icon"
        >
          <!--v-if-->
          <!--v-if-->
        </div>
        <div
          class="t-cell__title"
        >
          多行标题
          <!--v-if-->
          <div
            class="t-cell__description"
          >
            一段很长很长的内容文字
          </div>
        </div>
        <!--v-if-->
        <!--v-if-->
      </div>
      <div
        class="t-cell t-cell--middle t-cell--bordered"
      >
        <div
          class="t-cell__left-icon"
        >
          
          <svg
            class="t-icon t-icon-app"
            fill="none"
            height="1em"
            viewBox="0 0 16 16"
            width="1em"
          >
            <path
              d="M11.25 1.75a3 3 0 100 6 3 3 0 000-6zm-2 3a2 2 0 114 0 2 2 0 01-4 0zM2 3a1 1 0 011-1h3.5a1 1 0 011 1v3.5a1 1 0 01-1 1H3a1 1 0 01-1-1V3zm1 0v3.5h3.5V3H3zM2 9.5a1 1 0 011-1h3.5a1 1 0 011 1V13a1 1 0 01-1 1H3a1 1 0 01-1-1V9.5zm1 0V13h3.5V9.5H3zM8.5 9.5a1 1 0 011-1H13a1 1 0 011 1V13a1 1 0 01-1 1H9.5a1 1 0 01-1-1V9.5zm1 3.5H13V9.5H9.5V13z"
              fill="currentColor"
              fill-opacity="0.9"
            />
          </svg>
          
          <!--v-if-->
        </div>
        <div
          class="t-cell__title"
        >
          多行带图标
          <!--v-if-->
          <div
            class="t-cell__description"
          >
            一段很长很长的内容文字
          </div>
        </div>
        <!--v-if-->
        <div
          class="t-cell__right-icon"
        >
          <svg
            class="t-icon t-icon-chevron-right"
            fill="none"
            height="1em"
            viewBox="0 0 16 16"
            width="1em"
          >
            <path
              d="M6.46 12.46l-.92-.92L9.08 8 5.54 4.46l.92-.92L10.92 8l-4.46 4.46z"
              fill="currentColor"
              fill-opacity="0.9"
            />
          </svg>
        </div>
      </div>
      <div
        class="t-cell t-cell--middle t-cell--bordered"
      >
        <div
          class="t-cell__left-icon"
        >
          
          <div
            class="t-avatar t-avatar--circle"
          >
            <div
              class="t-avatar__inner"
            >
              <div
                class="t-image t-image--round"
              >
                <div
                  class="t-image__status"
                >
                  <svg
                    class="t-icon t-icon-ellipsis"
                    fill="none"
                    height="1em"
                    viewBox="0 0 16 16"
                    width="1em"
                  >
                    <path
                      d="M3 9a1 1 0 110-2 1 1 0 010 2zM7 8a1 1 0 102 0 1 1 0 00-2 0zM12 8a1 1 0 102 0 1 1 0 00-2 0z"
                      fill="currentColor"
                      fill-opacity="0.9"
                    />
                  </svg>
                </div>
                <img
                  alt=""
                  class="t-image__img"
                  src="https://tdesign.gtimg.com/mobile/demos/avatar_1.png"
                  style="object-fit: fill; object-position: center;"
                />
              </div>
            </div>
            <!--v-if-->
          </div>
          
          <!--v-if-->
        </div>
        <div
          class="t-cell__title"
        >
          多行带头像
          <!--v-if-->
          <div
            class="t-cell__description"
          >
            一段很长很长的内容文字
          </div>
        </div>
        <!--v-if-->
        <div
          class="t-cell__right-icon"
        >
          <svg
            class="t-icon t-icon-chevron-right"
            fill="none"
            height="1em"
            viewBox="0 0 16 16"
            width="1em"
          >
            <path
              d="M6.46 12.46l-.92-.92L9.08 8 5.54 4.46l.92-.92L10.92 8l-4.46 4.46z"
              fill="currentColor"
              fill-opacity="0.9"
            />
          </svg>
        </div>
      </div>
      <div
        class="t-cell t-cell--middle t-cell--bordered"
      >
        <div
          class="t-cell__left-icon"
        >
          <!--v-if-->
          
          <img
            class="t-cell__image"
            src="https://tdesign.gtimg.com/mobile/%E5%9B%BE%E7%89%87.png"
          />
          
        </div>
        <div
          class="t-cell__title"
        >
          多行带图片
          <!--v-if-->
          <div
            class="t-cell__description"
          >
            一段很长很长的内容文字
          </div>
        </div>
        <!--v-if-->
        <!--v-if-->
      </div>
      <div
        class="t-cell t-cell--middle t-cell--bordered"
      >
        <div
          class="t-cell__left-icon"
        >
          <!--v-if-->
          <!--v-if-->
        </div>
        <div
          class="t-cell__title"
        >
          多行标题
          <!--v-if-->
          <div
            class="t-cell__description"
          >
            一段很长很长的内容文字，长文本自动换行，该选项的描述是一段很长的内容
          </div>
        </div>
        <!--v-if-->
        <!--v-if-->
      </div>
      <div
        class="t-cell t-cell--middle t-cell--bordered"
      >
        <div
          class="t-cell__left-icon"
        >
          <!--v-if-->
          <!--v-if-->
        </div>
        <div
          class="t-cell__title"
        >
          多行高度不定，长文本自动换行，该选项的描述是一段很长的内容
          <!--v-if-->
          <div
            class="t-cell__description"
          >
            一段很长很长的内容文字，长文本自动换行，该选项的描述是一段很长的内容
          </div>
        </div>
        <!--v-if-->
        <!--v-if-->
      </div>
      
    </div>
  </div>
</div>
`;

exports[`Cell > Cell singleVue demo works fine 1`] = `
<div>
  <div
    class="t-cell-group"
  >
    <!--v-if-->
    <div
      class="t-cell-group__container border--top-bottom"
    >
      
      <div
        class="t-cell t-cell--middle t-cell--bordered"
      >
        <div
          class="t-cell__left-icon"
        >
          <!--v-if-->
          <!--v-if-->
        </div>
        <div
          class="t-cell__title"
        >
          单行标题
          <!--v-if-->
          <!--v-if-->
        </div>
        <!--v-if-->
        <!--v-if-->
      </div>
      <div
        class="t-cell t-cell--middle t-cell--bordered"
      >
        <div
          class="t-cell__left-icon"
        >
          <!--v-if-->
          <!--v-if-->
        </div>
        <div
          class="t-cell__title"
        >
          单行标题
          <span
            class="t-cell--required"
          >
             *
          </span>
          <!--v-if-->
        </div>
        <!--v-if-->
        <!--v-if-->
      </div>
      <div
        class="t-cell t-cell--middle t-cell--bordered"
      >
        <div
          class="t-cell__left-icon"
        >
          <!--v-if-->
          <!--v-if-->
        </div>
        <div
          class="t-cell__title"
        >
          单行标题
          <!--v-if-->
          <!--v-if-->
        </div>
        <div
          class="t-cell__note"
        >
          辅助信息
        </div>
        <!--v-if-->
      </div>
      <div
        class="t-cell t-cell--middle t-cell--bordered"
      >
        <div
          class="t-cell__left-icon"
        >
          <!--v-if-->
          <!--v-if-->
        </div>
        <div
          class="t-cell__title"
        >
          单行标题
          <!--v-if-->
          <!--v-if-->
        </div>
        <!--v-if-->
        <div
          class="t-cell__right-icon"
        >
          <svg
            class="t-icon t-icon-chevron-right"
            fill="none"
            height="1em"
            viewBox="0 0 16 16"
            width="1em"
          >
            <path
              d="M6.46 12.46l-.92-.92L9.08 8 5.54 4.46l.92-.92L10.92 8l-4.46 4.46z"
              fill="currentColor"
              fill-opacity="0.9"
            />
          </svg>
        </div>
      </div>
      <div
        class="t-cell t-cell--middle t-cell--bordered"
      >
        <div
          class="t-cell__left-icon"
        >
          <!--v-if-->
          <!--v-if-->
        </div>
        <div
          class="t-cell__title"
        >
          单行标题
          <!--v-if-->
          <!--v-if-->
        </div>
        <div
          class="t-cell__note"
        >
          辅助信息
        </div>
        <div
          class="t-cell__right-icon"
        >
          <svg
            class="t-icon t-icon-chevron-right"
            fill="none"
            height="1em"
            viewBox="0 0 16 16"
            width="1em"
          >
            <path
              d="M6.46 12.46l-.92-.92L9.08 8 5.54 4.46l.92-.92L10.92 8l-4.46 4.46z"
              fill="currentColor"
              fill-opacity="0.9"
            />
          </svg>
        </div>
      </div>
      <div
        class="t-cell t-cell--middle t-cell--bordered"
      >
        <div
          class="t-cell__left-icon"
        >
          <!--v-if-->
          <!--v-if-->
        </div>
        <div
          class="t-cell__title"
        >
          单行标题
          <!--v-if-->
          <!--v-if-->
        </div>
        <div
          class="t-cell__note"
        >
          
          <div
            class="t-badge"
          >
            <div
              class="t-badge__inner t-badge--medium t-badge--circle"
            >
              2
            </div>
            <!---->
          </div>
          
        </div>
        <div
          class="t-cell__right-icon"
        >
          <svg
            class="t-icon t-icon-chevron-right"
            fill="none"
            height="1em"
            viewBox="0 0 16 16"
            width="1em"
          >
            <path
              d="M6.46 12.46l-.92-.92L9.08 8 5.54 4.46l.92-.92L10.92 8l-4.46 4.46z"
              fill="currentColor"
              fill-opacity="0.9"
            />
          </svg>
        </div>
      </div>
      <div
        class="t-cell t-cell--middle t-cell--bordered"
      >
        <div
          class="t-cell__left-icon"
        >
          <!--v-if-->
          <!--v-if-->
        </div>
        <div
          class="t-cell__title"
        >
          单行标题
          <!--v-if-->
          <!--v-if-->
        </div>
        <div
          class="t-cell__note"
        >
          
          <span
            class="t-switch t-is-checked"
          >
            <!--v-if-->
            <span
              class="t-switch__node"
              style=""
            />
          </span>
          
        </div>
        <!--v-if-->
      </div>
      <div
        class="t-cell t-cell--middle t-cell--bordered"
      >
        <div
          class="t-cell__left-icon"
        >
          <svg
            class="t-icon t-icon-app"
            fill="none"
            height="1em"
            viewBox="0 0 16 16"
            width="1em"
          >
            <path
              d="M11.25 1.75a3 3 0 100 6 3 3 0 000-6zm-2 3a2 2 0 114 0 2 2 0 01-4 0zM2 3a1 1 0 011-1h3.5a1 1 0 011 1v3.5a1 1 0 01-1 1H3a1 1 0 01-1-1V3zm1 0v3.5h3.5V3H3zM2 9.5a1 1 0 011-1h3.5a1 1 0 011 1V13a1 1 0 01-1 1H3a1 1 0 01-1-1V9.5zm1 0V13h3.5V9.5H3zM8.5 9.5a1 1 0 011-1H13a1 1 0 011 1V13a1 1 0 01-1 1H9.5a1 1 0 01-1-1V9.5zm1 3.5H13V9.5H9.5V13z"
              fill="currentColor"
              fill-opacity="0.9"
            />
          </svg>
          <!--v-if-->
        </div>
        <div
          class="t-cell__title"
        >
          单行标题
          <!--v-if-->
          <!--v-if-->
        </div>
        <!--v-if-->
        <!--v-if-->
      </div>
      
    </div>
  </div>
</div>
`;
